lib/book.rb

```diff
@@ -0,0 +1,17 @@
+class Book
+  attr_reader :title, :author, :rented
+  attr_accessor :issue_count
+
+  def initialize(data)
+    @title = data[:title]
+    @author = data[:author_first_name] + " " + data[:author_last_name]
+    @publication_date = data[:publication_date]
+    @rented = nil
+    @issue_count = 0
```

Comment:
We wouldn't want to store this info on the book since the same book might be in 10 different libraries, we wouldn't want it's rented status from one library to affect it's rented status at another library. The library itself should hold onto that info. :)